files/stream.js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "stream.js",
+  "repo": "dionyziz/stream.js",
+  "files": {
+    "include": ["stream.js", "stream.min.js", "stream.min.js.map"]
```

Comment:
I don't see the min or map files https://github.com/dionyziz/stream.js/tree/v1.0.1